nemo/collections/nlp/modules/common/megatron/language_model.py

```diff
@@ -559,7 +562,9 @@ def __init__(
             assert 0 < rotary_percentage <= 1
             if rotary_percentage < 1:
                 rotary_dim = int(rotary_dim * rotary_percentage)
-            self.rotary_pos_emb = RotaryEmbedding(rotary_dim)
+            self.rotary_pos_emb = RotaryEmbedding(
```

Comment:
maybe not in this PR. we need to add seq_len_interpolation_factor for all the models that uses RoPe.